src/unit_tests/wazuh_modules/vulnerability_detector/test_wm_vuln_detector.c

```diff
@@ -319,6 +319,22 @@ void test_pkg_version_relate_epoch_LT(void **state)
     assert_int_equal(ret, 0);
 }
 
+void test_pkg_version_relate_epoch_negative_nvd(void **state)
+{
+    struct pkg_version *version_a = state[0];
+    struct pkg_version *version_b = state[1];
+
+    version_a->epoch = -1;
+    version_a->version = "1";
```

Comment:
When running tests over `valgrind` it reports many errors related to Invalid writes, like this one:

```
==52637== Invalid write of size 8
==52637==    at 0x163AC7: test_pkg_version_relate_epoch_negative_nvd (test_wm_vuln_detector.c:328)
==52637==    by 0x591C360: cmocka_run_one_test_or_fixture (in /usr/local/lib/libcmocka.so.0.7.0)
==52637==    by 0x591D49E: _cmocka_run_group_tests (in /usr/local/lib/libcmocka.so.0.7.0)
==52637==    by 0x168EEE: main (test_wm_vuln_detector.c:1262)
==52637==  Address 0x6335618 is 0 bytes after a block of size 8 alloc'd
==52637==    at 0x4C31B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==52637==    by 0x1625EE: setup_versions (test_wm_vuln_detector.c:31)
==52637==    by 0x591C6E9: cmocka_run_one_test_or_fixture.constprop.10 (in /usr/local/lib/libcmocka.so.0.7.0)
==52637==    by 0x591D46D: _cmocka_run_group_tests (in /usr/local/lib/libcmocka.so.0.7.0)
==52637==    by 0x168EEE: main (test_wm_vuln_detector.c:1262)
```

It can be solved by changing the lines 31 and 32 to:
```
os_calloc(1, sizeof(struct pkg_version), version_a);
os_calloc(1, sizeof(struct pkg_version), version_b);
```

Reply:
DONE